apps/common/views.py

```diff
@@ -0,0 +1,51 @@
+from rest_framework.permissions import AllowAny
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from rest_framework import status
+
+from apps.orders.models import Order
+
+from apps.common.helpers import stripe
+from config.settings import env
+
+status_mapping = {
+    'payment_intent.succeeded': Order.Status.CONFIRMED,
+
+    'payment_intent.canceled': Order.Status.CANCELED,
+    'payment_intent.payment_failed': Order.Status.CANCELED,
+}
+
+
+class StripeWebhookView(APIView):
+    permission_classes = (AllowAny,)
+
+    def post(self, request, *args, **kwargs):
```

Comment:
primul lucru cand primesti date in webhook -> salveaza loguri cu tot ce ai primit